pkgs/applications/science/biology/truvari/default.nix

```diff
@@ -1,42 +1,69 @@
 { lib
 , fetchFromGitHub
 , python3Packages
+, runtimeShell
+, htslib
 }:
 
-python3Packages.buildPythonApplication rec {
+let
+  ssshtest = fetchFromGitHub {
+    owner = "ryanlayer";
+    repo = "ssshtest";
```

Comment:
what kind of owner / repo is this?

The original repository seems to have been removed. I don't use this tool anymore, maybe we should just remove it?

It seems to have moved here:

https://github.com/ACEnglish/truvari

but I can not confirm that.

Reply:
That's just a bash script used to run tests.

Reply:
Thanks, really nice work! 😊